sql/updates/world/6.x/2016_04_21_00_world.sql

```diff
@@ -0,0 +1,43 @@
+-- Gate of the Setting Sun
+UPDATE instance_template SET script='instance_gate_setting_sun' WHERE map=962;
+UPDATE creature_template SET ScriptName='mob_serpent_spine_defender' WHERE entry=58146;

```

Comment:
For future reference, when doing the cleanup of the scripts, be sure to change `mob_` prefixes to `npc_`.
Overall, great effort. I'm glad you decided to separate one massive PR into smaller ones, one per instance. +1